.github/workflows/aks.yaml

```diff
@@ -230,3 +230,12 @@ jobs:
           description: Connectivity test cancelled
           state: pending
           target_url: ${{ env.check_url }}
+
+      - name: Send slack notification
+        if: ${{ (cancelled() || failure()) && (github.event_name == 'schedule' || github.event_name == 'push') }}
```

Comment:
I think we need something more fine-grained for `cancelled()` here (or we need to remove it). Now that we merged https://github.com/cilium/cilium/pull/16199, all outdated GitHub jobs will be cancelled (for example when we push a new version of a PR's branch). That will generated a lot of Slack notifications.

Reply:
@pchaigno this bit `(github.event_name == 'schedule' || github.event_name == 'push')` will only generate notifications for failing/cancelled builds that run on schedule, or on merge (`push` event). I did this so we are able to monitor closely what state the master branch (and possibly the stable branches) are in. No PR failure will cause a notification, this should be monitored by the PR author on his own.

Reply:
Makes sense :+1: